commit d6b91a20c8284a2b3502986e187194805500eb8b (refs/changes/33/2633/3)
Author: Brittany Wald <brittany.wald@joyent.com>
Date:   2017-09-22T10:28:20-07:00 (2 years, 1 month ago)
    
    MANTA-3382 node-fash data structure exposure
    Reviewed by: Richard Bradley <richard.bradley@joyent.com>
    Approved by: Richard Bradley <richard.bradley@joyent.com>

diff --git a/package.json b/package.json
index 44e78c8..dd75ce4 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "bunyan-syslog": "0.2.2",
         "clone": "0.1.9",
         "dtrace-provider": "0.2.8",
-        "fash": "2.4.1",
+        "fash": "2.5.0",
         "fast": "0.3.8",
         "ldapjs": "0.6.3",
         "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#db3efc1",
